packages/inter-protocol/test/auction/test-proportionalDist.js

```diff
@@ -49,7 +58,12 @@ const checkProportions = (
   const expectedXfer = [];
   for (let i = 0; i < rawDeposits.length; i += 1) {
     const seat = harden({});
-    deposits.push({ seat, amount: collateral.make(rawDeposits[i]) });
+    const { collect } = rawDeposits[i];
+    deposits.push({
+      seat,
+      amount: collateral.make(BigInt(rawDeposits[i].deposit)),
+      goal: collect ? currency.make(BigInt(collect)) : null,
```

Comment:
`collect` was to match what I had proposed for the property of the `deposit` element type. To match the type now:
```suggestion
      goal: goal ? currency.make(BigInt(goal)) : null,
```